src/modules/data-ingestion/product/product.processor.ts

```diff
@@ -448,19 +454,34 @@ export class ProductProcessor implements OnApplicationBootstrap {
             await this.nobleService.addTimedLogMessage(task, '--- Chunk finished ---');
         }
 
-        //todo: delete products that are not in the file
-        if ((bulkOperationRecord.explicitIds ?? []).length === 0) {
-            //this was a full sync, so we can delete all ids we did not see
-            //deleteAllExcept
-        } else {
-            //this was a partial sync, so we need to delete all ids that we did not see
+        console.log('allProductShopifyIdsFromThisFile', allProductShopifyIdsFromThisFile);
+        if (runFullSyncLogic) {
```

Comment:
It's probably also worth updating the "lastSafetySync" time on the retailer entity here, I think that is still missing from safety syncs